qemu/tests/chardev_pci_serial_login.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from virttest import error_context, env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify the login function of pci-serial (RHEL and x86 only):
+    1) Start guest with pci-serial with backend unix_socket
+    2) nc host/path for unix and tcp backend.
+    3) login guest and create file, move files
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    serial_id = params.objects('serials')[-1]
+    params['start_vm'] = 'yes'
+    for backend in ['tcp_socket', 'unix_socket']:
+        params['chardev_backend_%s' % serial_id] = backend
+        env_process.preprocess_vm(test, params, env, params['main_vm'])
+        vm = env.get_vm(params["main_vm"])
+        vm.verify_alive()
+        error_context.context("login guest with %s" % backend, logging.info)
+        session = vm.wait_for_login()
```

Comment:
At least `wait_for_serial_login`, please check if it use the pci-serial.

Reply:
Thanks and update. besides, we don't support TCP option while using `wait_for_serial_login` to login guest and I think we don't need to change the framework since we just need to verify that the pci_serial can log in the guest. What do you think?

Reply:
@nanliu-r  It depends how you design the test case in polarion. I do not have strong idea with this.